### PR TITLE
Tandy/section 8

### PR DESCRIPTION
**Pull request by Tandy-Nicole**
Requesting for a review thanks!

### CODE REVIEW

File: client/src/pages/Home/MemberSays/Cards.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		Content:
+			"“This group has given me the courage to go against the system and fight to get my daughter the help needs.”",
+	},
+
+	{
+		Header: "SENsational Life",
+		Content:
+			"“So happy that our closed parent support group has such an impact. Come on and over join us.”",
+	},
+];
```
Comment (khaledkz) [inline client/src/pages/Home/MemberSays/Cards.js:22]: can you move cardContent array to helper file.

File: client/src/pages/Home/MemberSays/Cards.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+export default function Cards() {
+	return (
+		<div className="grid-container">
+			{cardContent.map((x) => (
+				<div className="grid-item">
+					<h4>{x.Header}</h4>
+					<p>{x.Content}</p>
```
Comment (khaledkz) [inline client/src/pages/Home/MemberSays/Cards.js:30]: can you use better name than x for ex card or item

File: client/src/pages/Home/MemberSays/Member.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	font-size: 30px;
+	text-align: center;
+}
+.button:hover {
+	box-shadow: 0 12px 16px 0 rgba(0, 0, 0, 0.24),
+		0 17px 50px 0 rgba(0, 0, 0, 0.19);
+	background-color: grey;
+	color: black;
+}
+.button {
```
Comment (khaledkz) [inline client/src/pages/Home/MemberSays/Member.css:22]: can you improve the class name i think button is not a good class name.

File: client/src/pages/Home/MemberSays/MemberSays.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+		<div
+			style={{
+				backgroundColor: "#1E6DD0",
+				display: "flex",
+				flexDirection: "column",
+				alignContent: "center",
+				height: "75vh",
+				alignItems: "stretch",
+				gap: "40px",
+				padding: "5px",
```
Comment (khaledkz) [inline client/src/pages/Home/MemberSays/MemberSays.js:16]: it's much better to give the div a class name then you can remove this inline style to the style file

**Review by khaledkz — COMMENTED** [review]
Looks good! added a couple of comments

File: client/src/pages/Home/MemberSays/Cards.js (hunk 1 above)
Comment (Tandy-Nicole) [reply client/src/pages/Home/MemberSays/Cards.js:22]: Done

File: client/src/pages/Home/MemberSays/Cards.js (hunk 2 above)
Comment (Tandy-Nicole) [reply client/src/pages/Home/MemberSays/Cards.js:30]: Used item. Done

File: client/src/pages/Home/MemberSays/Member.css (hunk 3 above)
Comment (Tandy-Nicole) [reply client/src/pages/Home/MemberSays/Member.css:22]: Changed to register-btn. Done

File: client/src/pages/Home/MemberSays/MemberSays.js (hunk 4 above)
Comment (Tandy-Nicole) [reply client/src/pages/Home/MemberSays/MemberSays.js:16]: Done

**Review by youssef910 — APPROVED** [review]

**Review by khaledkz — APPROVED** [review]

**Pull request merged**